docs/building/windows-core.md

```diff
@@ -18,8 +18,7 @@ This guide assumes that you have recursively cloned the PowerShell repository an
 
 ### Visual Studio
 
-You will need to install an edition of Visual Studio 2015 (Community, Enterprise, or Professional) with the optional feature 'Common Tools for Visual C++' installed.
-The free Community edition of Visual Studio 2015 can be downloaded [here](https://www.visualstudio.com/visual-studio-community-vs/).
+This repository requires at least Visual Studio 2019 16.6 Preview 2. The free Community edition of Visual Studio can be downloaded from [Microsoft](https://visualstudio.microsoft.com/downloads/).
```

Comment:
I think it makes no sense to mention "preview"
```suggestion
This repository requires at least Visual Studio 2019 16.6. The free Community edition of Visual Studio can be downloaded from [Microsoft](https://visualstudio.microsoft.com/downloads/).
```

Reply:
[PowerShell-Native](https://github.com/PowerShell/PowerShell-Native) is a seperate repo which is not included here as a git submodule, so it would seem MSVC is not required?

Reply:
PowerShell Native was decoupled to simplify build process. But really same developers work on both PowerShell and PowerShell native.

Reply:
@TravisEz13 would you like to weigh in?

Reply:
My strong opinion is only for `Preview 2` - I'd not want to update this every week :-)

Reply:
the dependancy is `Preview 2` **or later**, the current version is `Preview 6`